From: YueHaibing <yuehaibing@huawei.com>
Date: Thu, 22 Aug 2019 22:43:46 +0800
Subject: ACPI: SBS: remove unused const variable 'SMBUS_PEC'
Patch-mainline: v5.4-rc1
Git-commit: 85a5f06773c85f8ec5030ed65e5fa3190b3f16bd
References: jsc#SLE-16407

drivers/acpi/sbshc.h:18:17: warning:
 SMBUS_PEC defined but not used [-Wunused-const-variable=]

SMBUS_PEC is never used since introduction in
commit 91087dfa51a2 ("ACPI: SBS: Split host
controller (ACPI0001) from SBS driver (ACPI0002)"),
so just remove it.

Reported-by: Hulk Robot <hulkci@huawei.com>
Signed-off-by: YueHaibing <yuehaibing@huawei.com>
Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>
Acked-by: Lee, Chun-Yi <jlee@suse.com>
---
 drivers/acpi/sbshc.h |    2 --
 1 file changed, 2 deletions(-)

--- a/drivers/acpi/sbshc.h
+++ b/drivers/acpi/sbshc.h
@@ -15,8 +15,6 @@ enum acpi_smb_protocol {
 	SMBUS_BLOCK_PROCESS_CALL = 0xd,
 };
 
-static const u8 SMBUS_PEC = 0x80;
-
 enum acpi_sbs_device_addr {
 	ACPI_SBS_CHARGER = 0x9,
 	ACPI_SBS_MANAGER = 0xa,
